### DISCUSSION TITLE
Icon and Filter display issue

### DISCUSSION THREAD

**Discussion by IbrahimARahman** [LazyVim/LazyVim#1282] [Q&A]
Hey y'all just install LazyVim and having a couple issues. Find files icon doesn't show for some reason. (Have SauceCodePro Nerd Font Mono) Also the filter display works in neotree but not in normal files. Screenshots attatched
The first image just shows the LazyVim front page
second is after hitting / in neotree
third in after hitting / in init.lua buffer

<img width="1046" alt="image" src="https://github.com/LazyVim/LazyVim/assets/40009958/1686abf5-7c7a-4286-8124-e734e2a8d964">
<img width="1512" alt="image" src="https://github.com/LazyVim/LazyVim/assets/40009958/0ebe3767-84cb-44de-a54e-8455cd20818f">
<img width="1509" alt="image" src="https://github.com/LazyVim/LazyVim/assets/40009958/029ca505-e2f7-4ec2-9721-f04d43f1a795">

**Comment by dpetka2001** [comment]
Update your fonts to the latest nerd-fonts version. There have been breaking changes when they released v3.0

**Reply by IbrahimARahman** [reply]
Did that, same thing still happening, any solution for second issue as well?

**Comment by IbrahimARahman** [comment]
It also seems that when I hit specific keys it breaks the search in regular files, like u, instead of typing u in the search bar is does the u command (undo)

**Reply by IbrahimARahman** [reply]
Ok I realized the second issue was because I didnt know what leap.nvim was and I just have missing icons.

**Comment by IbrahimARahman** [comment]
Alright the issue is Nerd Fonts V3.0.0, supposedly they broke icons with that update, but I tried downloading an older version and they still don't work.

**Reply by dpetka2001** [reply]
Why download older version of fonts when Lazyvim supports newer version? Open your font manager, delete the older version you have and then install the new one. It's probably a problem with your font cache. If you're on Linux [this](https://medium.com/source-words/how-to-manually-install-update-and-uninstall-fonts-on-linux-a8d09a3853b0) is a good guide for how to update. If you're on another OS you would have to google that yourself.

**Reply by IbrahimARahman** [reply]
I updated again and cleared cache, no effect.

**Reply by dpetka2001** [reply]
Well it must be related to your terminal and how it applies the fonts. You should google for that.

**Reply by IbrahimARahman** [reply]
I use iTerm2, idk, ill just deal with broken fonts ig

**Reply by dpetka2001** [reply]
Can you try another terminal? Like kitty or wezterm? Just to check if the problem lies with your terminal or not. So you know what to search for.

**Reply by dustinfarris** [reply]
Make sure you have "use another font for non-ascii text" turned off in iTerm2

![image](https://github.com/LazyVim/LazyVim/assets/1087165/17cfadbb-de35-4b54-9888-e403fd6a9e68)

**Reply by PhenomenaPh** [reply]
Did you solve this problem because im having the same issue? Tried a big number of 'nerd' fonts and still 'find' button does not have an icond...

**Reply by ProHaller** [reply]
Same here, I'd be happy to have a solution for that.

**Reply by dpetka2001** [reply]
It's the code point of the font you're using that is not able to get rendered by iTerm2. See this [comment](https://github.com/LazyVim/LazyVim/issues/1139#issuecomment-1806858433) for how to get around that.

**Reply by ProHaller** [reply]
Thank you! I switched to kitty and that solved it too.
